server/jwt.go

```diff
@@ -161,6 +172,7 @@ func validateToken(jwtKeyFunc keyfunc.Keyfunc, token string, expectedTenantIDs [
 	for _, expectedTenantID := range expectedTenantIDs {
 		if claims["tid"] == expectedTenantID {
 			logger.Info("Validated token, and authorized request from matching tenant")
+			logger.Info("Claim tenant id: ", claims["tid"])
```

Comment:
Can we just add the `tid` to the logger context early on so all logs end up with this being annotated?

Reply:
Yeah, the `tid` is already in the logger context, this may have slipped while debugging.